README.md

```diff
@@ -8,25 +8,29 @@ The list of providers is published in the form of a statically hosted OPTiMaDe I
 
 If you specifically seek the current list of providers for the latest version of the OPTiMaDe specification, you can access it at this URL:
 
-- [https://www.optimade.org/providers/links/](https://www.optimade.org/providers/links/)
+- [https://www.optimade.org/providers/providers.json](https://www.optimade.org/providers/providers.json)
 
 If you seek the current list of providers for any older version of the OPTiMaDe specification, you can access it using this URL:
 
-- [https://www.optimade.org/providers/*\<version\>*/links/](https://www.optimade.org/providers/<version>/links/)
+- [https://www.optimade.org/providers/*\<version\>*/links](https://www.optimade.org/providers/<version>/links)
 
 
 ## Repository organization
 
 The OPTiMaDe providers repository is hosted here: [https://github.com/Materials-Consortia/providers](https://github.com/Materials-Consortia/providers)
 
-The paths in the repository are organized this way:
+The repository is organized this way:
 
-- `src/<version>/providers.json` is the current providers.json file formatted according to OPTiMaDe version `<version>`.
+- `/src/links/<version>/providers.json` is the current providers.json file formatted according to OPTiMaDe version `<version>` and any later version that uses a format that is backward compatible with this version.
 
-- `info` and `links` are symbolic links into the subdirectory matching the latest release of the OPTiMaDe specification.
+- `/src/links/<version>/info.json` is the proper response to the info endpoint formatted according to OPTiMaDe version `<version>` and any later version that uses a format that is backward compatible with this version.
 
-- `<version>`: every released version of the OPTiMaDe specification gets a subdirectory organized to confirm with the URLs as defined for an OPTiMaDe base url. The `<version>/links/index.html` is a symbolic link to `src/<version>/providers.json`.
+- `/_redirect` specify http rewrites to map index meta-database URLs `/<version>/info` and `/<version>/links` to the corresponding files under `src/`, as well as `/providers.json`.
+
+- `/<version>` directories contain symlinks `info.html` and `links.html` to the corresponding files under `src/` to somewhat support hosting solutions that do not understand the instructions in `_redirect` but which support "pretty URLs".
+
+- `/providers.json` is a symbolic link continously updated to point at the latest version of the providers.json file under `/src/` to somewhat support hosting solutions that do not understand the instructions in `_redirect`.
 
 To update the list of providers, please file a pull-request against the repository.
-This pull-request MUST update ALL the provider.json files `src/<version>/providers.json`.
-This way, the current list of providers is updated for implementations across all versions of the OPTiMaDe protocol.
+This pull-request MUST update ALL the provider.json files `src/links/<version>/providers.json`.
```

Comment:
This triggers a red alert for me: 

> pull-request MUST update ALL the provider.json files

 The need for such requirement indicates that SPOT/DRY/normalisation is violated, and information will eventually get out of sync.

I would suggest that all information about any provider is stored in one place, just once, and, if necessary, other files or links are auto-generated (e.g. by Make ;). This suggests a single `providers.{json,xml,yaml,whatever}` source somewhere, where each provider specifies its general information (e.g. the base URI), and then lists Optimade versions that it supports, possible with (relative) URLs.

Reply:
While possibly a good idea, this isn't something that this PR *changes* from how the repository presently is set up. So please post it as a separate issue, e.g., "Avoid redundacy across differently formatted providers files", and we can pick up the discussion there.